contents/docs/_snippets/groups.mdx

```diff
@@ -0,0 +1,327 @@
+> **Note: ** This is a paid feature and is not available on the open source or free cloud plan. See our [pricing page](/pricing) for more.
+
+**Groups** are a powerful feature in PostHog that aggregate events based on entities. They enable you to track trends, insights, and dashboards at an entity-level – as opposed to regular events, which only show you insights on a user-level.
+
+To clarify what we mean, let's look at a few examples:
+
+1. For B2B SaaS apps, you can create a *company* group type to aggregate events at a company-level. This enables you to calculate metrics such as `number of daily active companies`, `company churn rate`, or `how many companies have adopted a new feature`.
+
+2. For a communication app like Slack, grouping events at a channel-level enables you to measure metrics like `average number of messages per channel`, `number of monthly active channels`, or `total number of channel participants`.
+
+3. For a social media app like Twitter, events can be grouped at a tweet-level – enabling you to measure `average number of replies per tweet`, or `total count of unique tweeters per month`.
+
+> Groups are often confused with [cohorts](/docs/data/cohorts), but they each serve different purposes:
+>
+> * Groups aggregate events, and do not necessarily have to be connected to a user.
+> * Cohorts represent a specific set of users – e.g., a list of users that all belong to the same company.
+> 
+> Cohorts are easier to use and quicker to get started, whereas groups are more powerful. If your only goal is to create a list of **users** with something in common, we recommond cohorts instead of groups..
+
+## How to create groups
+
+Groups are created and defined in your code when sending events. 
+
+In the example below, we create a group type `company`. Then, for each company, we set the `group key` as the unique identifier for that specifc company. This can be anything that helps you identify it, such as ID or domain.
```

Comment:
```suggestion
In the example below, we create a group type `company`. Then, for each company, we set the group key as the unique identifier for that specific company. This can be anything that helps you identify it, such as ID, name, or domain.
```

Reply:
Name is not recommended, as mentioned in the call out below